go.mod

```diff
@@ -43,7 +43,7 @@ require (
 	github.com/hashicorp/go-retryablehttp v0.7.0
 	github.com/hashicorp/go-tfe v0.19.1-0.20211020175229-e52963e079d0
 	github.com/hashicorp/go-uuid v1.0.2
-	github.com/hashicorp/go-version v1.2.1
+	github.com/hashicorp/go-version v1.3.0
```

Comment:
For the benefit of other potential reviewers: this bump seems to just introduce the `version.Core` method, from hashicorp/go-version#85, and so shouldn't lead to any behavior changes for other parts of Terraform not using that new method.